src/components/common/Tile/PersonTile/styled.jsx

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 208px;
+  height: auto;
+  padding: 16px;
+  box-shadow: ${({ theme }) => theme.properties.tile.boxShadow};
+  gap: 12px;
+
+  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.s}) {
+  }
+`;
+
+export const ImageWrapper = styled.div`
+  border-radius: 5px;
+  width: 177px;
+  min-width: 177px;
+  /* height: 264px; */
+  min-height: 264px;
+  background-color: ${({ theme }) => theme.colors.tile.imageWrapper};
```

Comment:
Try to delete comments - its a good practice to clean code after testing styles.

Reply:
you're right, I didn't notice that. already corrected